### ISSUE TITLE
make:crud - error of naming twig varible in index method

### ISSUE THREAD

**Issue by s0lar** [symfony/maker-bundle#134]
My Entity name is "App\Entity\DepartmentPo" 
After make:crud in controller a get method 

```
App\Controller\DepartmentPoController.php

    public function index(DepartmentPoRepository $departmentPoRepository): Response
    {
        return $this->render('department_po/index.html.twig', ['departmentPos' => $departmentPoRepository->findAll()]);
    }
```

```
templates\department_po\index.html.twif

    {% for department_po in department_pos %}
```

In the controller defines "departmentPos" varible but in twig we trying find "department_pos". 
I think the problem in the entity name "DepartmentPo".

**Comment by weaverryan** [comment]
Will be fixed in #132 :)

**Comment by weaverryan** [comment]
Let us know if you have any other issues!
